x11-libs/libdlo/libdlo-0.1.2.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools
+
+DESCRIPTION="A fully open source driver which supports all DisplayLink devices"
+HOMEPAGE="https://libdlo.freedesktop.org/wiki/"
+SRC_URI="https://people.freedesktop.org/~berniet/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="static-libs test-program"
+
+RDEPEND="virtual/libusb:0="
+
+DEPEND="${RDEPEND}"
+
+DOCS=( "AUTHORS" "ChangeLog" "Guide-v104.pdf" "README" )
+
+src_prepare() {
+	default
+
+	# AM_PROG_AR must be defined or automake will fail with:
+	# archiver requires 'AM_PROG_AR' in 'configure.ac'.
+	sed -i -e '/AC_PROG_CC/a AM_PROG_AR' configure.ac || die
+
+	# Only build the Displaylink test program, if a user want's it.
+	# If a user want's it, rename test1 to displaylink-test.
+	if ! use test-program; then
+		sed -i -e '/test\/Makefile/d' configure.ac || die
+		sed -i -e 's/src.*/src/' Makefile.am || die
+		sed -i -e '/test$/d' Makefile.am || die
+		sed -i -e '/test\/images.*/d' Makefile.am || die
+		sed -i -e '/TESTS .*/d' Makefile.am || die
+	else
+		sed -i -e 's/TESTS .*/TESTS = test\/displaylink-test/' Makefile.am || die
```

Comment:
Fine for me. Just did it this way, as someone told me, that I should avoid using things like mv in src_install and patch the build system.

Reply:
ah, that depends, but this case is trivial. again, a build system patch that you can take upstream is always preferred, so that these workarounds can at some point be dropped.

Reply:
Okay, changed now in newest version.